Content.Server/DropPod/DropPodSystem.cs

```diff
@@ -0,0 +1,342 @@
+                Thread.Sleep(component.Time * 1000);
+            }
+
+            if (!TryComp<MapGridComponent>(end_station_uid, out var end_station_gridComp))
+                return;
+
+            var xform = Transform(end_station_uid);
+            Random rnd = new Random();
+            while (true)
+            {
+                int radius = rnd.Next(-5, 6);
+                var x_coord = (int) (Coords.X - 0.5f) + radius; // This way we create a spread on landing to make it more interesting (spread within 5 tiles).
+                radius = rnd.Next(-5, 6);
+                var y_coord = (int) (Coords.Y - 0.5f) + radius;
+
+                var tile = new Vector2i(x_coord, y_coord);
+                if (_atmosphere.IsTileSpace(end_station_uid, xform.MapUid, tile)) // the point from which the drop pod generation starts should not be a space tile...
+                {
+                    continue; // if this is the case, then we are looking for the next possible point
+                }
+
+                var pos = _mapSystem.GridTileToLocal(end_station_uid, end_station_gridComp, tile);
+                if (!TryComp<MapGridComponent>(start_station_uid, out var start_station_gridComp))
+                    return;
+
+                CheckTils(start_station_uid, start_station_gridComp, end_station_uid, end_station_gridComp, pos);
+                break;
+            }
+        }
+        #endregion
+
+        #region moving the capsule
+        /// <summary>
+        /// This method iterates through all tiles within our borders (which are set by beacons) and creates similar ones (objects are transferred, not created) at the point of grounding
+        /// </summary>
+        /// <param name="start_station_uid"> it is needed to identify an object that is located in a certain tile at the Nuke Ops station </param>
+        /// <param name="start_grid"> it is needed to determine the type of tile at the Nuke Ops station </param>
+        /// <param name="end_station_uid"> it is necessary to understand where to move objects </param>
+        /// <param name="end_grid"> it is needed to determine the local coordinates at the station where the disembarkation will take place </param>
+        /// <param name="pos"> it is necessary to move the object to the desired coordinates </param>
+        private void CheckTils(EntityUid start_station_uid, MapGridComponent start_grid, EntityUid end_station_uid, MapGridComponent end_grid, EntityCoordinates pos)
+        {
+            left_border -= 0.5f;  // we are aligning the values of the borders, because now we are counting
+            right_border -= 0.5f; // from the center of the tile (on which the lighthouse stands), and not from the lower-left corner
+            top_border -= 0.5f;
+            bottom_border -= 0.5f;
+
+            for (int i = (int) left_border + 1; i < (int) right_border; i++)
```

Comment:
All of this should look extremely sus, and I don't think such things should be in the content of individual systems. It clearly requires engine work to add support for combining grids on single coordinates